client/src/observatory/results.tsx

```diff
@@ -0,0 +1,668 @@
+        <span className="host">{hostAsRedirectChain(host, result)}</span>
+      </h2>
+      <section className="scan-result">
+        <section className="grade-trend">
+          <div className="overall">
+            <span
+              aria-label="show info tooltip"
+              className="info-tooltip"
+              tabIndex={0}
+            >
+              <div
+                className={`grade grade-${result.scan.grade?.[0]?.toLowerCase()}`}
+              >
+                {fixMinusSymbol(result.scan.grade)}
+              </div>
+              <Tooltip>
+                <table className="grade-tooltip">
+                  <thead>
+                    <tr>
+                      <th>Grade</th>
+                      <th>Score</th>
+                    </tr>
+                  </thead>
+                  <tbody>
+                    {scoringTable.map((st) => {
+                      return (
+                        <tr
+                          className={
+                            result.scan.grade === st.grade ? "current" : ""
+                          }
+                          key={st.grade}
+                        >
+                          <td>{fixMinusSymbol(st.grade)}</td>
+                          <td>
+                            {st.scoreText}{" "}
+                            {result.scan.grade === st.grade && st.stars && (
+                              <StarsSVG />
+                            )}
+                          </td>
+                        </tr>
+                      );
+                    })}
+                  </tbody>
+                </table>
+              </Tooltip>
+            </span>
+          </div>
+          {trend(result)}
+        </section>
+        <section className="data">
+          <div>
+            <a href="docs/scoring">
+              <span className="label">Score:</span>
+            </a>{" "}
+            {result.scan.score}/100
+          </div>
+          <div>
+            <a href="#scan_history">
+              <span className="label">Scan Time:</span>
+            </a>{" "}
+            {new Date(result.scan.scanned_at).toLocaleString([], {
+              dateStyle: "medium",
+              timeStyle: "medium",
+            })}
+          </div>
+          <a href="docs/scoring##tests-and-score-modifiers">
+            <span className="label">Tests Passed:</span>
+          </a>{" "}
+          {result.scan.tests_passed}/{result.scan.tests_quantity}
+        </section>
+        <section className="actions">
+          {!isServer && (
+            <CountdownButton
+              host={host}
+              from={result.scan.scanned_at}
+              duration={60}
+              title="Rescan"
+              onClickHandler={rescanTrigger}
+            />
+          )}
+          <div className="scan-another">
+            <InternalLink
+              to="../"
+              onClick={() => gleanClick(`${OBSERVATORY}: click: scan-another`)}
```

Comment:
I'd prefer to keep scan-another because it describes the click better. We do not have the term `new` used anywhere in observatory. Removed the `click:`.